src/List.tsx

```diff
@@ -1,40 +1,51 @@
-import Item from "./Item";
+import { ListItem } from "./App";
+import {makeStyles} from 'tss-react'
+
+interface ListProps {
+  title: string;
+  items: Array<ListItem>;
+  handleCheck: (item: ListItem, checked: boolean) => void;
+}
+
+const useStyles = makeStyles()(() => ({
+  container: {
+    display: "flex",
+    flexDirection: "column",
+  }
+}))
+
+function List({ title, items, handleCheck }: ListProps) {
+  const { classes } = useStyles();
 
-function List({ title, items, handleCheck }) {
   return (
-    <div
-      style={{
-        display: "flex",
-        flexDirection: "column",
-      }}
-    >
+    <div className={classes.container}>
+      <h2 style={{ color: "red", textAlign: "center" }}>
+        {title}
+      </h2>
+      <p style={{ textAlign: "center" }}>
+        Number of items: {items.length}{" "}
+      </p>
```

Comment:
I moved this to top of the component, because it looks better for UI/UX purposes. (More visible)

Reply:
You're right, makes sense